### DISCUSSION TITLE
Memory issues are getting progressively worst

### DISCUSSION THREAD

**Discussion by Rainz3** [lshqqytiger/stable-diffusion-webui-directml#39] [General]
I'm not sure what has changed, I used to be able to do up to batch sizes of up to 4 and use controlnet canny with batch size of 2. Now, after generating just a few images I can't use canny at all and am limited to just 1 in batch size. Using --opt-sub-quad-attention --no-half --precision full --medvram --disable-nan-check --sub-quad-q-chunk-size 512 --sub-quad-kv-chunk-size 512 --sub-quad-chunk-threshold 80. I have to constantly restart stable-diffusion just to get anything done. This includes having lowvram checked in controlnet.

Also, please is there any way to retain settings when closing the browser. I've lost countless hours accidentally closing the tab or refreshing the page.

This is on windows 11, 5700xt, 5800x, 32gb ram

**Comment by lshqqytiger** [comment]
You don't need to close the browser tab. WebUI will automatically reconnect with server.

**Reply by Rainz3** [reply]
> You don't need to close the browser tab. WebUI will automatically reconnect with server.

Yeah thats more so just for myself. I don't know what the deal is. I did a full reinstall and now I can do batch sizes up to 5, way better than before. But now I can't even use canny without the lowvram checked which wasn't the case. Same with Openpose. I used to be able to use that without issue but now I can't, even with lowvram checked. Using same options as above.
